cuebot/src/main/java/com/imageworks/spcue/VirtualProc.java

```diff
@@ -127,18 +127,12 @@ public static final VirtualProc build(DispatchHost host, DispatchFrame frame) {
                 proc.coresReserved = wholeCores * 100;
             } else {
                 if (frame.threadable) {
-                    // The following condition checks to see if the service being invoked is arnold. If it is, take up all the remaining
-                    // cores.
-                    if ("arnold".equals(frame.services)) {
+                    if (host.idleMemory - frame.minMemory
+                            <= Dispatcher.MEM_STRANDED_THRESHHOLD) {
                         proc.coresReserved = wholeCores * 100;
-                    } else {
-                        if (host.idleMemory - frame.minMemory
-                                <= Dispatcher.MEM_STRANDED_THRESHHOLD) {
-                            proc.coresReserved = wholeCores * 100;
-                        }
-                        else {
-                            proc.coresReserved = getCoreSpan(host, frame.minMemory);
-                        }
+                    }
+                    else {
```

Comment:
nit: combine this line and the line above.

Reply:
picked